highs/interfaces/highs_c_api.h

```diff
@@ -2350,6 +2377,39 @@ HighsInt Highs_feasibilityRelaxation(void* highs,
                                      const double* local_upper_penalty,
                                      const double* local_rhs_penalty);
 
+/**
+ * Attempt to compute an irreducible infeasibility subsystem (IIS) for
+ * an LP, QP, or the relaxation of a MIP. If no IIS is found, then the
+ * number of IIS columns and rows will be zero.
+ *
+ * @param highs                      A pointer to the Highs instance.
+ * @param const HighsInt iis_num_col Number of columns in the IIS.
+ * @param const HighsInt iis_num_row Number of rows in the IIS.
+ * @param const HighsInt* col_index  An array of length [iis_num_col], to be
+ *                                   filled with the indices of original
+ *                                   variables in the IIS.
+ * @param const HighsInt* row_index  An array of length [iis_num_col], to be
+ *                                   filled with the indices of original
+ *                                   constraints in the IIS.
+ * @param const HighsInt* col_bound  An array of length [iis_num_col], to be
+ *                                   filled with the bound status of variables
+ *                                   in the IIS.
+ * @param const HighsInt* row_bound  An array of length [iis_num_col], to be
+ *                                   filled with the bound status of constraints
+ *                                   in the IIS.
+ * @param const HighsInt* col_status An array of length [num_col], to be
+ *                                   filled with the IIS status of all original
+ *                                   variables.
+ * @param const HighsInt* row_status n array of length [num_col], to be
+ *                                   filled with the IIS status of all original
+ *                                   constraints.
+ *
+ * @returns A `kHighsStatus` constant indicating whether the call succeeded.
+ */
+HighsInt Highs_getIis(void* highs, HighsInt* iis_num_col, HighsInt* iis_num_row,
+                      HighsInt* col_index, HighsInt* row_index,
+                      HighsInt* col_bound, HighsInt* row_bound,
+                      HighsInt* col_status, HighsInt* row_status);
```

Comment:
What is the difference between `_bound` and `_status`? The `_bound` is free/lower/upper/boxed and the `_status` is in conflict/not in conflcit/maybe in conflict?

If we're returning only a subset of the rows and columns, the status would never be "not in conflict", so are both vectors needed?

Perhaps to answer my own question: if the IIS is not minimal, then there might be a "maybe in conflict" entry. If all are "in conflict" then the IIS is minimal.

Reply:
> What is the difference between `_bound` and `_status`? The `_bound` is free/lower/upper/boxed and the `_status` is in conflict/not in conflcit/maybe in conflict?

Yes
 
> If we're returning only a subset of the rows and columns, the status would never be "not in conflict", so are both vectors needed?

These vectors aren't needed at the moment, but if the IIS calculation is interrupted by timeout, say, then (to answer your own question) the IIS is not minimal, so there might be a "maybe in conflict" entry. If all are "in conflict" then the IIS is minimal.